go-controller/pkg/ovn/controller/apbroute/network_client.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 )
 
+const (
```

Comment:
I think the best option would be to make it a member of northBoundClient like
```
type northBoundClient struct {
    controllerName string
}
```
and then pass the controller name from `NewExternalMasterController` similar to the previous `parentControllerName` variable